backend/tracim_backend/lib/utils/authentication.py

```diff
@@ -168,6 +188,8 @@ def _idp_chooser(self, request: TracimRequest) -> Response:
         )
 
     def _acs(self, request: TracimRequest) -> Response:
```

Comment:
what is acs? I'd like some explanation of what it implements (the name is not self-explicit). Maybe link to related specification

Reply:
ACS is a term specific to SAML, however I could indeed add some comments above to explain

Reply:
fixed